react-app/src/utils/leaguehelpers.js

```diff
@@ -74,10 +81,48 @@ export default {
 
 	convertMatchTime: (time) => {
 		//This is just for the schedules bc theyre always at night
+		//we can change it later..
 		let newTime = (parseInt(time) - 1200).toString();
 		return newTime.substr(0,1) +':' + newTime.substr(1,2) + 'pm';
 	},
 
-	
+	getMatchTimes: (league, dateId) => {
+
+		if(league === null || dateId === ''){
+			return null;
+		}
+
+		let matchTimes = [];
+
+		league.scheduledMatches.forEach((match) =>{
+			if(match.dateId === dateId){
+				matchTimes.push(match.matchTime);
+			}
+		});
+
+		return matchTimes;
+
+	},
+
+	getMatches: (league, dateId) => {
+		if(league == null || dateId == ''){
+			return null;
+		}
+
+		let matches = [];
+
+		league.scheduledMatches.forEach((match) =>{
```

Comment:
I think this would make more sense as a .reduce. I'd say this should be sorted by match time as well but I suspect this function will need to be reworked to filter on matchtime so don't worry about it.

Reply:
Sorry not reduce, map is what you want